.github/workflows/zip.yml

```diff
@@ -657,8 +657,8 @@ jobs:
       matrix:
         artifact: [Firebase-actions-dir, Firebase-actions-dir-dynamic]
         build-env:
-          - os: macos-14
-            xcode: Xcode_15.2
+          # - os: macos-14
+          #   xcode: Xcode_15.2
```

Comment:
nit: Should these be deleted instead of commented out?

Reply:
Updating to keep them commented but point to Xcode 16.3.

Reply:
SGTM. Let's consider switching the current ones to `macos-14` / `Xcode_16.1` when we uncomment the macOS 15 / Xcode 16.3 ones.

Reply:
Yes, I though I think you means `macos-14 / Xcode_16.2` since the binaries are built with .2